src/NCDO/CDO.cs

```diff
@@ -635,8 +635,7 @@ public void Sort()
             request.Success = response.IsSuccessStatusCode;
             request.ResponseMessage = response;
             //https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Transfer-Encoding
-            if ((response.Headers.TransferEncodingChunked.HasValue && response.Headers.TransferEncodingChunked.Value) ||
-                response.Content.Headers.ContentLength > 0)
+            if (response.IsSuccessStatusCode)
```

Comment:
This will not garantee that there is data in the response.

Reply:
why check for data in the response ?